manifests/check.pp

```diff
@@ -107,11 +107,18 @@
 #   Set this to 'absent' to remove it completely.
 #   Default: undef
 #
+# [*content*]
```

Comment:
They're different configuration scopes.  Mailer is up at top-level (the same as "checks").  Custom adds key/value pairs two levels deep within the checks.mycheck scope.